pkg/internal/validate/validate_abort_action.go

```diff
@@ -0,0 +1,19 @@
+package validate
+
+import (
+	"fmt"
+
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/wdk"
+)
+
+func ValidAbortActionArgs(args *wdk.AbortActionArgs) error {
+	if args == nil {
+		return fmt.Errorf("args cannot be nil")
+	}
+
+	if args.Reference == nil {
```

Comment:
Check for empty string.

Hint:
```suggestion
	if is.BlankString(args.Reference) {
```

Reply:
as `Reference` is now type `primitives.Base64String` changed to:
```
if args.Reference == "" {
```